src/Storage/Field/Collection/FieldCollection.php

```diff
@@ -146,16 +146,6 @@ public function offsetGet($offset)
     }
 
     /**
-     * Overrides the default array access method and passes on to the get method
-     * @param mixed $offset
-     * @return mixed
-     */
-    public function offsetGet($offset)
```

Comment:
@rossriley what @CarsonF said … it was my bad in the first place, for bigger branch merges I usually run tests before committing the merge, but you know that time you drop process 'cause "It'll be fiiiiiiiiiiiiiiiiiiiine!"

Admittedly it isn't best practice and I can move to a new PR in a few hours when I get up for real. My motivation is that @CarsonF is away for the week, and my time zones sucks too coordinate merges, and I was super excited about this PR. 

Not meant to excuse, just an explanation :smile: